g11n-ws/build.gradle

```diff
@@ -69,7 +69,7 @@ subprojects{
     	commonsCodecVersion='1.15'
     	
     	guavaVersion="30.1.1-jre"
-    	snakeyam="1.29"
+    	snakeyam="1.31"
```

Comment:
I check maven repo today, the latest version is 1.32, please change it to the latest.